wiki/en/Running-a-Server.md

```diff
@@ -36,84 +37,102 @@ However, various problems can also arise when setting up Servers - especially wh
+For information, [see the Directories guide](Directories).
 
-If you want to run a Server on a Raspberry Pi (or a different armhf/arm64 debian-based device), you will need to download the `.deb` files for 32 bit `armhf` or 64 bit `arm64`, not the default `amd64` ones you'd use on an Intel/AMD based machine.
+---
 
+## Configuration options
 
-### Installation
+Depending on your operating system and how you are running the server, you can set Server options and make them persistent between reloads as follows:
 
-1. Download the [latest headless (amd64) .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }}) or, if you use a Raspberry Pi etc. download the [latest armhf .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }}) or the [latest arm64 .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-arm64 }})
-1. Update apt to make sure you have a current list of standard packages: `sudo apt update`
-1. Install the Jamulus package: `sudo apt install ./{{ site.download_file_names.deb-headless }}` or for RasPi etc. armhf: `sudo apt install ./{{ site.download_file_names.deb-headless-armhf }}`; arm64: `sudo apt install ./{{ site.download_file_names.deb-headless-arm64 }}`
-1. Enable the headless Server process via systemd: `sudo systemctl enable jamulus-headless`
-1. Add your desired [command line options](Running-a-Server#command-line-options) to the `ExecStart` line in the systemd service file by running `sudo systemctl edit --full jamulus-headless` (By default you will be running an Unregistered Server).
-1. Reload the systemd files `sudo systemctl daemon-reload` and restart the headless Server: `sudo systemctl restart jamulus-headless`
-1. Check all is well with `systemctl status jamulus-headless` (hit `q` to get back to the command prompt).
+**Linux headless mode** (Debian/Ubuntu using systemd) 
 
-You can control Jamulus with the `systemctl` command. For example, to stop the Server cleanly:
+Add your desired command line options to the `ExecStart` line in the systemd service file by running `sudo systemctl edit --full jamulus-headless`
 
-`sudo systemctl stop jamulus-headless`
+**GUI mode** (all platforms)
 
-_To upgrade your Server, just repeat the steps above._
+Any settings made using the graphical interface will be stored in the `Jamulusserver.ini` file (do not attempt to edit this file by hand). However, some options are not available via the GUI and need to be set via the command line. For more information, see [Servers on the desktop](#servers-on-the-desktop)
 
-### Configuration
+---
 
-#### Running in Registered mode
+### Server mode-related options
 
-The following minimum setup is required to [run a Registered Server](Running-a-Server#server-types):
+##### `-e or --directoryserver` 
```

Comment:
Must, at some point, get changed to say `--directory`... and consign `--directoryserver` to be another deprecated option...